Mac/MainWindow/MainWindowController.swift

```diff
@@ -269,6 +269,12 @@ class MainWindowController : NSWindowController, NSUserInterfaceValidations {
 		openArticleInBrowser(sender)
 	}
 
+	@IBAction func openInBrowserAlternate(_ sender: Any?) {
```

Comment:
Would you change the function name to `openInBrowserUsingOppositeOfSettings` — thanks!

Reply:
Updated ✅